operations/productcategories.md

```diff
@@ -0,0 +1,123 @@
+# Product categories
+
+## Get all product categories
+
+Returns all categories of products. This operation uses [Pagination](../guidelines/pagination.md).
```

Comment:
I would like ALL operations that now support pagination AND multi-property to say "Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../guidelines/multi-property.md)." If the operation just supports multi-property, it should say "Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).".

Reply:
It's published now :-D